hera_pspec/testing.py

```diff
@@ -103,7 +103,10 @@ def build_vanilla_uvpspec(beam=None):
         integration_array[s] = np.ones((Nblpairts, Npols), dtype=np.float)
         nsample_array[s] = np.ones((Nblpairts, Npols), dtype=np.float)
         window_function_array[s] = np.ones((Nblpairts, Ndlys, Ndlys, Npols), dtype=np.float64)
-        cov_array[s] = np.moveaxis(np.array([[np.identity(Ndlys,dtype=np.complex) \
+        cov_array_real[s] = np.moveaxis(np.array([[np.identity(Ndlys,dtype=np.float) \
```

Comment:
I'm not sure I see the prudence in splitting up our covariances into a real and imaginary part (and reworking all of our code to support this). uvp.data_array is complex. To me, the most economical solution is to just make the covariances of these complex arrays also complex.

Reply:
OK. Now I realize that a single complex array is not enough information to describe the error bars on both the real and imaginary component of a power-spectrum estimate, essentially because there are 2 x (N+1) x N / 2 degrees of freedom for the imag/real covariances but a single complex covariance only has (N-1) x N / 2 + N degrees of freedom (since the diagonal has to be real). I think it does make sense to move to real/imag arrays. Thank you @JianrongTan / @acliu for explaining this.